<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="10" failed="2" ignored="0" total="17" passed="5">
  <reporter-output>
    <line>
      <![CDATA[Read the email and password]]>
    </line>
    <line>
      <![CDATA[The email is Valid]]>
    </line>
    <line>
      <![CDATA[send login request]]>
    </line>
    <line>
      <![CDATA[Read the receiver data, subject and body]]>
    </line>
    <line>
      <![CDATA[<a href='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\composeEmail_18_05_2018_11_45_23.png'> <img src='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\composeEmail_18_05_2018_11_45_23.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\openEmail_18_05_2018_11_45_25.png'> <img src='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\openEmail_18_05_2018_11_45_25.png' height='100' width='100'/> </a>]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="41659" started-at="2018-05-18T20:44:44Z" finished-at="2018-05-18T20:45:26Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="41659" started-at="2018-05-18T20:44:44Z" finished-at="2018-05-18T20:45:26Z">
      <class name="scenarios.EmailAutomation">
        <test-method status="PASS" signature="setUp()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="setUp" is-config="true" duration-ms="9238" started-at="2018-05-18T20:44:44Z" finished-at="2018-05-18T20:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="readAuthData()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="readAuthData" duration-ms="18" started-at="2018-05-18T20:44:53Z" description="Read the email and password of the sender and store them in variables" finished-at="2018-05-18T20:44:53Z">
          <reporter-output>
            <line>
              <![CDATA[Read the email and password]]>
            </line>
          </reporter-output>
        </test-method> <!-- readAuthData -->
        <test-method status="PASS" signature="enterEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="enterEmail" duration-ms="1768" started-at="2018-05-18T20:44:53Z" description="Enter the email and press next." depends-on-methods="scenarios.EmailAutomation.readAuthData" finished-at="2018-05-18T20:44:55Z">
          <reporter-output>
            <line>
              <![CDATA[The email is Valid]]>
            </line>
          </reporter-output>
        </test-method> <!-- enterEmail -->
        <test-method status="PASS" signature="enterPasword()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="enterPasword" duration-ms="1467" started-at="2018-05-18T20:44:55Z" description="enter the password" depends-on-methods="scenarios.EmailAutomation.enterEmail" finished-at="2018-05-18T20:44:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterPasword -->
        <test-method status="PASS" signature="logIn()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="logIn" duration-ms="2419" started-at="2018-05-18T20:44:57Z" description="Send Login Request" depends-on-methods="scenarios.EmailAutomation.enterPasword" finished-at="2018-05-18T20:44:59Z">
          <reporter-output>
            <line>
              <![CDATA[send login request]]>
            </line>
          </reporter-output>
        </test-method> <!-- logIn -->
        <test-method status="PASS" signature="getEmailData()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="getEmailData" duration-ms="4" started-at="2018-05-18T20:44:59Z" description="Read the email reciever, subject and body." depends-on-methods="scenarios.EmailAutomation.logIn" finished-at="2018-05-18T20:44:59Z">
          <reporter-output>
            <line>
              <![CDATA[Read the receiver data, subject and body]]>
            </line>
          </reporter-output>
        </test-method> <!-- getEmailData -->
        <test-method status="FAIL" signature="composeEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="composeEmail" duration-ms="24167" started-at="2018-05-18T20:44:59Z" description="Open new email window and fill the text inputs" depends-on-methods="scenarios.EmailAutomation.getEmailData" finished-at="2018-05-18T20:45:23Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for all conditions to be valid: presence of element located by: By.xpath: //*[contains(text(), 'COMPOSE')] && visibility of element located by By.xpath: //*[contains(text(), 'COMPOSE')] && element to be clickable: By.xpath: //*[contains(text(), 'COMPOSE')] (tried for 20 second(s) with 500 milliseconds interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for all conditions to be valid: presence of element located by: By.xpath: //*[contains(text(), 'COMPOSE')] && visibility of element located by By.xpath: //*[contains(text(), 'COMPOSE')] && element to be clickable: By.xpath: //*[contains(text(), 'COMPOSE')] (tried for 20 second(s) with 500 milliseconds interval)
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:82)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at utility.HelperFunctions.getButton(HelperFunctions.java:27)
at pageObjects.HomePage.composeButton(HomePage.java:13)
at modules.SendEmail.newEmail(SendEmail.java:27)
at scenarios.EmailAutomation.composeEmail(EmailAutomation.java:123)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.xpath: //*[contains(text(), 'COMPOSE')]
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.10.0', revision: '176b4a9', time: '2018-03-02T19:03:16.397Z'
System info: host: 'DESKTOP-DEJDQ69', ip: '10.100.19.128', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:896)
at java.util.Optional.orElseThrow(Unknown Source)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:895)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:44)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:183)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:180)
at org.openqa.selenium.support.ui.ExpectedConditions$48.apply(ExpectedConditions.java:1470)
at org.openqa.selenium.support.ui.ExpectedConditions$48.apply(ExpectedConditions.java:1466)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
            <line>
              <![CDATA[<a href='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\composeEmail_18_05_2018_11_45_23.png'> <img src='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\composeEmail_18_05_2018_11_45_23.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- composeEmail -->
        <test-method status="FAIL" signature="openEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="openEmail" duration-ms="6" started-at="2018-05-18T20:45:25Z" description="Open the specified email. and assert the subject" depends-on-methods="scenarios.EmailAutomation.getEmailData" finished-at="2018-05-18T20:45:25Z">
          <exception class="org.openqa.selenium.NoSuchSessionException">
            <message>
              <![CDATA[Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.10.0', revision: '176b4a9', time: '2018-03-02T19:03:16.397Z'
System info: host: 'DESKTOP-DEJDQ69', ip: '10.100.19.128', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.10.0', revision: '176b4a9', time: '2018-03-02T19:03:16.397Z'
System info: host: 'DESKTOP-DEJDQ69', ip: '10.100.19.128', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:127)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:398)
at org.openqa.selenium.remote.RemoteWebDriver.findElementsByXPath(RemoteWebDriver.java:477)
at org.openqa.selenium.By$ByXPath.findElements(By.java:356)
at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:359)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:895)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:44)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:183)
at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:180)
at org.openqa.selenium.support.ui.ExpectedConditions$48.apply(ExpectedConditions.java:1470)
at org.openqa.selenium.support.ui.ExpectedConditions$48.apply(ExpectedConditions.java:1466)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
at utility.HelperFunctions.getButton(HelperFunctions.java:27)
at pageObjects.HomePage.emailFromList(HomePage.java:61)
at scenarios.EmailAutomation.openEmail(EmailAutomation.java:217)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchSessionException -->
          <reporter-output>
            <line>
              <![CDATA[<a href='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\openEmail_18_05_2018_11_45_25.png'> <img src='F:\bbbbbbbbbbbbbbbbbbbbbb\EmailTest\failure_screenshots\openEmail_18_05_2018_11_45_25.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- openEmail -->
        <test-method status="SKIP" signature="addAttachment()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="addAttachment" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Attach the file to the email." depends-on-methods="scenarios.EmailAutomation.composeEmail" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.addAttachment()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.addAttachment()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addAttachment -->
        <test-method status="SKIP" signature="validateBody()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="validateBody" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Validate the Body." depends-on-methods="scenarios.EmailAutomation.openEmail" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.validateBody()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.validateBody()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateBody -->
        <test-method status="SKIP" signature="sendEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="sendEmail" duration-ms="1" started-at="2018-05-18T20:45:26Z" description="Send the email" depends-on-methods="scenarios.EmailAutomation.addAttachment" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.sendEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.sendEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendEmail -->
        <test-method status="SKIP" signature="validateFrom()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="validateFrom" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Validate the email of the sender." depends-on-methods="scenarios.EmailAutomation.validateBody" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.validateFrom()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.validateFrom()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateFrom -->
        <test-method status="SKIP" signature="Logout()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="Logout" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Logging out" depends-on-methods="scenarios.EmailAutomation.sendEmail" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.Logout()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.Logout()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Logout -->
        <test-method status="SKIP" signature="readRecepientAuthData()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="readRecepientAuthData" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Read the email and password of the sender and store them in variables" depends-on-methods="scenarios.EmailAutomation.Logout" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.readRecepientAuthData()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.readRecepientAuthData()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readRecepientAuthData -->
        <test-method status="SKIP" signature="enterRecepientEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="enterRecepientEmail" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Enter the recepient email and press next." depends-on-methods="scenarios.EmailAutomation.readRecepientAuthData" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.enterRecepientEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.enterRecepientEmail()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterRecepientEmail -->
        <test-method status="SKIP" signature="enterRecepientPasword()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="enterRecepientPasword" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="enter the recepient password" depends-on-methods="scenarios.EmailAutomation.enterRecepientEmail" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.enterRecepientPasword()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.enterRecepientPasword()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterRecepientPasword -->
        <test-method status="SKIP" signature="logInRecepient()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="logInRecepient" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Send recepient Login Request" depends-on-methods="scenarios.EmailAutomation.enterRecepientPasword" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.logInRecepient()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.logInRecepient()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logInRecepient -->
        <test-method status="SKIP" signature="getRecepientEmailData()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="getRecepientEmailData" duration-ms="0" started-at="2018-05-18T20:45:26Z" description="Get the email data from the excel file." depends-on-methods="scenarios.EmailAutomation.logInRecepient" finished-at="2018-05-18T20:45:26Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method EmailAutomation.getRecepientEmailData()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method EmailAutomation.getRecepientEmailData()[pri:0, instance:scenarios.EmailAutomation@5f341870] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getRecepientEmailData -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:scenarios.EmailAutomation@5f341870]" name="afterClass" is-config="true" duration-ms="0" started-at="2018-05-18T20:45:26Z" finished-at="2018-05-18T20:45:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- scenarios.EmailAutomation -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
